### ISSUE TITLE
Exclude tables from access

### ISSUE THREAD

**Issue by cpdean** [Squarespace/pgbedrock#17]
Say we had a schema, `the_data` with three tables, `a`, `b`, and `c`.  We have some users that have read/write access to all the tables in `the_data`, but after a while we decide that we do not like table `b` so we'd like to deprecate it by first restricting access to it for a while and then dropping the table properly later.

I'm open to other suggestions like "how about you just move table `b` to an off-limits schema", but I wanted to discuss if this is a pattern that's useful and if so what would that look like in the spec def?

**Comment by zcmarine** [comment]
@cboline's suggestion is that we have a list of objects that pgbedrock will just ignore, so that's one possibility.

**Comment by cpdean** [comment]
ohh that's a good call.  then we don't have to check weird interactions, we can just mark stuff to be ignored and manually manage it out of band with the spec.

**Comment by cpdean** [comment]
yeah i guess we either rename this or open a new ticket for that feature request
